ops/terraform/services/server/server-regression/lambda-src/spice-trigger/spice-trigger.py

```diff
@@ -0,0 +1,44 @@
+import boto3
+from datetime import datetime
+import os
+
+ENV = os.environ.get("ENV", "")
+
+def lambda_handler(event, context):
+    if not ENV:
+        print("ENV was not defined, exiting...")
+        return
+
+    client = boto3.client('quicksight')
+    account_id = boto3.client('sts').get_caller_identity().get('Account')
+    data_set_id = get_data_set_id(client, account_id)
+    refresh_data_set(client, account_id, data_set_id)
+
+def get_data_set_id(client, account_id):
+    data_sets = client.search_data_sets(AwsAccountId=account_id, Filters=[
+        {
+            'Operator': 'StringLike',
+            'Name': 'DATASET_NAME',
+            'Value': f'bfd-{ENV}-server-regression'
+        },
+    ], MaxResults=100)
+
+    for data_set in data_sets["DataSetSummaries"]:
+        if data_set["Name"] == f"bfd-{ENV}-server-regression" and data_set["ImportMode"] == "SPICE":
+            return data_set["DataSetId"]
+    raise RuntimeError('QuickSight Dataset not found.')
+
+def refresh_data_set(client, account_id, data_set_id):
+    print('Refreshing SPICE data')
+    time_stamp = datetime.now().strftime("%Y%m%d-%H%M%S")
+    ingestion_id = time_stamp + '-' + data_set_id
+
+    try:
+        response = client.create_ingestion(AwsAccountId=account_id, DataSetId=data_set_id, IngestionId=ingestion_id,
+                                           IngestionType='INCREMENTAL_REFRESH')
+        ingestion_status = response["IngestionStatus"]
+        print(f'Refresh status of dataset {data_set_id} with an ingestion id of {ingestion_id} is {ingestion_status}')
+
+    except Exception as e:
+        print(e)
+        raise
```

Comment:
```suggestion
    response = client.create_ingestion(AwsAccountId=account_id, DataSetId=data_set_id, IngestionId=ingestion_id,
                                       IngestionType='INCREMENTAL_REFRESH')
    ingestion_status = response["IngestionStatus"]
    print(f'Refresh status of dataset {data_set_id} with an ingestion id of {ingestion_id} is {ingestion_status}')
```

I recommend removing the `try`/`except` clause here as the Python Lambda Runtime will print the full `Exception` and stack trace details when any uncaught `Exception`s are raised to it. So, you don't need to worry about `print`ing the `Exception` here.